src/main/java/org/spongepowered/api/item/recipe/Recipe.java

```diff
@@ -47,29 +44,10 @@
 public interface Recipe {
 
     /**
-     * Returns the list of item types that result when successful crafting of
-     * this Recipe is completed.
+     * Gets the result of this {@link Recipe}.
      *
-     * @return The resultant list of item types
+     * @return the results of this {@link Recipe}
      */
-    List<ItemType> getResultTypes();
```

Comment:
Likewise with the previous deprecated method, this should be left in with `@Deprecated` to use the `getResults()` list.